modules/ztp-creating-a-validator-inform-policy.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-creating-a-validator-inform-policy_{context}"]
+= Creating a validator inform policy
+
+Use the following procedure to create a validator inform policy for single node clusters,
+three-node clusters, and standard clusters.
+
+.Procedure
+
+. Create a standalone `PolicyGenTemplates` custom resource (CR) that contains the source file
+`validatorCRs/informDuValidator.yaml`.
+You only need one standalone `PolicyGenTemplates` CR for each  cluster type.
++
+* Single node clusters:
++
+[source,yaml]
+----
+group-du-sno-validator-ranGen.yaml
+apiVersion: ran.openshift.io/v1
+kind: PolicyGenTemplate
+metadata:
+  name: "group-du-sno-validator" <1>
+  namespace: "ztp-group" <2>
+spec:
+  bindingRules:
+    group-du-sno: "" <3>
+  bindingExcludedRules:
+    ztp-done: "" <4>
+  mcp: "master" <5>
+  sourceFiles:
+    - fileName: validatorCRs/informDuValidator.yaml
+      remediationAction: inform <6>
+      policyName: "du-policy" <7>
+----
+
+* Three-node clusters:
++
+[source,yaml]
+----
+group-du-3node-validator-ranGen.yaml
+apiVersion: ran.openshift.io/v1
+kind: PolicyGenTemplate
+metadata:
+  name: "group-du-3node-validator" <1>
+  namespace: "ztp-group" <2>
+spec:
+  bindingRules:
+    group-du-3node: "" <3>
+  bindingExcludedRules:
+    ztp-done: "" <4>
+  mcp: "master" <5>
+  sourceFiles:
+    - fileName: validatorCRs/informDuValidator.yaml
+      remediationAction: inform <6>
+      policyName: "du-policy" <7>
+----
+
+* Standard clusters:
++
+[source,yaml]
+----
+group-du-standard-validator-ranGen.yaml
+apiVersion: ran.openshift.io/v1
+kind: PolicyGenTemplate
+metadata:
+  name: "group-du-standard-validator" <1>
+  namespace: "ztp-group" <2>
+spec:
+  bindingRules:
+    group-du-standard: "" <3>
+  bindingExcludedRules:
+    ztp-done: "" <4>
+  mcp: "worker" <5>
+  sourceFiles:
+    - fileName: validatorCRs/informDuValidator.yaml
+      remediationAction: inform <6>
+      policyName: "du-policy" <7>
+----
+<1> The name of `PolicyGenTemplates` object. This name is also used as part of the names for the placementBinding, placementRule and policy are created in the requested `namespace`.
```

Comment:
Second sentence is kind of garbled, and I think you probably want to put the values in literal text, e.g., `placementBinding`, `placementRule` and `policy`? Also passive voice. Maybe, "When creating their names, the `placementBinding`, `placementRule` and `policy` incorporate this name in the requested `namespace`." I'm not clear on the meaning, but it could be worded better.

Reply:
Clarified this sentence.